cmake/kokkos_tribits.cmake

```diff
@@ -148,15 +148,20 @@ ELSE()
   CMAKE_PARSE_ARGUMENTS(PARSE
     ""
     ""
-    "SOURCES;CATEGORIES"
+    "SOURCES;CATEGORIES;ARGS"
     ${ARGN})
   VERIFY_EMPTY(KOKKOS_ADD_EXECUTABLE_AND_TEST ${PARSE_UNPARSED_ARGUMENTS})
   KOKKOS_ADD_TEST_EXECUTABLE(${ROOT_NAME}
     SOURCES ${PARSE_SOURCES}
   )
+  IF(DEFINED PARSE_ARGS)
```

Comment:
I actually don't know the answer. Is this undefined or is it just empty if no arguments are given?

Reply:
According to the [doc](https://cmake.org/cmake/help/v3.10/command/cmake_parse_arguments.html) it will be undefined.